Library/Formula/libdecnumber.rb

```diff
@@ -0,0 +1,41 @@
+class Libdecnumber < Formula
+  desc "ANSI C implementation of General Decimal Arithmetic"
+  homepage "http://speleotrove.com/decimal/#decNumber"
+  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
+  version "3.68"
+  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
+  resource "cmakelists" do
```

Comment:
I'm trying to achieve us being able to support this package without actually becoming the upstream maintainers or adding in unsupported, unknown-by-upstream build steps that if we get bug reports on we'll have to handle ourselves.

You should submit the CMakeList to upstream and try to talk them into using it, rather than working around upstream's lack of convenient build process on a package-manager-by-package-manager basis.